order-routing/javascript/rankers/default/src/index.liquid

```diff
@@ -0,0 +1,55 @@
+{%- if flavor contains "vanilla-js" -%}
```

Comment:
I notice eslint is not present in dev dependencies in package.json - are you able to confirm that the generated JS/TS is compliant with shopify's usual linter configuration?